### PR TITLE
feat : KBD component

### PR DESCRIPTION
**Pull request by MateoWartelle**
- [X] I have followed the [Your First Code Contribution section of the Contributing guide](https://github.com/themesberg/flowbite-react/blob/main/CONTRIBUTING.md#your-first-code-contribution)

Summarize the changes made and the motivation behind them.

Kbd component to match existing flowbite library

Reference related issues using `#` followed by the issue number.

If there are breaking API changes - like adding or removing props, or changing the structure of the theme - describe them, and provide steps to update existing code.

### CODE REVIEW

**Comment by MateoWartelle** [comment]
I am going to write some tests and it should be ready for review after. I am open to change whatever is suggested. 

thanks

**Comment by rluders** [comment]
@MateoWartelle  please, do not use merge, use rebase instead.

File: app/docs/components/kbd/kbd.mdx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          <Kbd icon={MdKeyboardArrowLeft}></Kbd>
+          <Kbd icon={MdKeyboardArrowRight}></Kbd>
+        </Table.Cell>
+        <Table.Cell>Choose and activate previous/next tab.</Table.Cell>
+      </Table.Row>
+    </Table.Body>
+  </Table>
+</CodePreview>
+
+## Arrow keys
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:82]: I think the arrows are different from the flowbite documentation, aren't they? Is there any particular reason?

File: app/data/components.tsx [hunk 2]
```diff
@@ -328,14 +328,14 @@ export const COMPONENTS_DATA: Component[] = [
...
-  //   name: 'KBD (Keyboard)',
-  //   image: '/images/components/keyboard.svg',
-  //   imageDark: '/images/components/keyboard-dark.svg',
-  //   link: `/docs/components/kbd`,
-  //   classes: 'w-40'
-  // },
+  {
+    id: '42',
+    name: 'KBD (Keyboard)',
+    image: '/images/components/keyboard.svg',
```
Comment (rluders) [inline app/data/components.tsx:334]: Both images, default and dark are broken. You will need to add them to: https://github.com/themesberg/flowbite-react/tree/main/public/images/components

**Review by rluders — CHANGES_REQUESTED** [review]

File: app/docs/components/kbd/page.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import type { Metadata, NextPage } from 'next';
+import KbdPageContent from '.';
+
+export const metadata: Metadata = {
+  description:
+    'Use the Kbd component as an inline element to denote textual user input from the keyboard inside paragraphs, tables, and other components',
+  title: 'React Forms - Flowbite',
```
Comment (SutuSebastian) [inline app/docs/components/kbd/page.tsx:7]: Is the title correct in this one?

File: app/docs/components/kbd/index.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import type { FC } from 'react';
+import { DocsContentLayout } from '~/app/components/docs-content-layout';
+import KbdDocs from './kbd.mdx';
+
+const FormsPageContent: FC = () => {
+  return (
+    <DocsContentLayout
+      title="React Kbd - Flowbite"
```
Comment (rluders) [inline app/docs/components/kbd/index.tsx:10]: React KBD (Keyboard) - Flowbite

File: app/docs/components/kbd/kbd.mdx [hunk 5]
```diff
@@ -0,0 +1,177 @@
+import { CodePreview } from '~/app/components/code-preview';
+import { Kbd, Table, theme } from '~/src';
+import { MdKeyboardArrowLeft, MdKeyboardArrowRight, MdKeyboardArrowDown, MdKeyboardArrowUp } from 'react-icons/md';
+
+The Kbd (Keyboard) component can be used to indicate a textual user input from the keyboard inside other elements such as in text, tables, cards, and more.
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:5]: The KDB (Keyboard)

File: app/docs/components/kbd/kbd.mdx [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+import { Kbd } from 'flowbite-react';
+```
+
+## Table of Contents
+
+## Default KBD
+
+Here’s a list of KBD components that you can use inside any other element.
+
+<CodePreview importFlowbiteReact="Kbd" title="Default Kbd">
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:23]: Default KDB

File: app/docs/components/kbd/kbd.mdx [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+  <Kbd>Esc</Kbd>
+  <Kbd>Spacebar</Kbd>
+  <Kbd>Enter</Kbd>
+</CodePreview>
+
+## KBD inside text
+
+Use this example by nesting an inline KBD component inside a paragraph.
+
+<CodePreview importFlowbiteReact="Kbd" title="Inside Text Kbd">
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:37]: Inside Text KDB

File: app/docs/components/kbd/kbd.mdx [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+</CodePreview>
+
+## KBD inside table
+
+The KBD component can also be used inside table components to denote what type of key can be pressed for certain descriptions.
+
+<CodePreview
+  importFlowbiteReact="Kbd, Table"
+  importExternal="import { MdKeyboardArrowLeft, MdKeyboardArrowRight, MdKeyboardArrowDown, MdKeyboardArrowUp  } from 'react-icons/md';"
+  title="Table Kbd"
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:48]: Table KDB

File: app/docs/components/kbd/kbd.mdx [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+</CodePreview>
+
+## Arrow keys
+
+Use this example to show arrow keys inside the KBD styled element.
+
+<CodePreview
+  importFlowbiteReact="Kbd"
+  importExternal="import { MdKeyboardArrowLeft, MdKeyboardArrowRight, MdKeyboardArrowDown, MdKeyboardArrowUp  } from 'react-icons/md';"
+  title="Arrow Keys Kbd"
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:89]: Arrow Keys KDB

File: app/docs/components/kbd/kbd.mdx [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+  <Kbd icon={MdKeyboardArrowDown}></Kbd>
+  <Kbd icon={MdKeyboardArrowLeft}></Kbd>
+  <Kbd icon={MdKeyboardArrowRight}></Kbd>
+</CodePreview>
+
+## Letter Keys
+
+Use this example to show arrow keys inside the KBD styled element.
+
+<CodePreview importFlowbiteReact="Kbd" title="Letter Keys Kbd">
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:101]: Letter Keys KDB

File: app/docs/components/kbd/kbd.mdx [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+    <Kbd>N</Kbd>
+    <Kbd>M</Kbd>
+  </div>
+</CodePreview>
+
+## Number Keys
+
+Use this example to show a key inside a KBD component from the english numeral system.
+
+<CodePreview importFlowbiteReact="Kbd" title="Number Keys Kbd">
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:135]: Number Keys KDB

File: app/docs/components/kbd/kbd.mdx [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+  <Kbd>8</Kbd>
+  <Kbd>9</Kbd>
+  <Kbd>0</Kbd>
+</CodePreview>
+
+## Function keys
+
+This example can be used to denote function keys inside the KBD component.
+
+<CodePreview importFlowbiteReact="Kbd" title="Function Keys Kbd">
```
Comment (rluders) [inline app/docs/components/kbd/kbd.mdx:152]: Function Keys KDB

File: app/docs/components/kbd/page.tsx [hunk 13]
```diff
@@ -0,0 +1,14 @@
+import type { Metadata, NextPage } from 'next';
+import KbdPageContent from '.';
+
+export const metadata: Metadata = {
+  description:
+    'Use the Kbd component as an inline element to denote textual user input from the keyboard inside paragraphs, tables, and other components',
```
Comment (rluders) [inline app/docs/components/kbd/page.tsx:6]: Use the KDB ....

File: app/docs/components/kbd/page.tsx (hunk 3 above)
Comment (rluders) [reply app/docs/components/kbd/page.tsx:7]: React KBD (Keyboard) - Flowbite

**Review by rluders — CHANGES_REQUESTED** [review]

**Comment by rluders** [comment]
Looks nice, @MateoWartelle - just some small details that need to be fixed and we are good to get it merged.

File: app/docs/components/kbd/kbd.mdx (hunk 1 above)
Comment (rluders) [reply app/docs/components/kbd/kbd.mdx:82]: 🤔 I think that the standard flowbite library is now using the flowbite icons, but we don't support it yet. So, just ignore it and leave it as it is for now.

**Pull request merged**